CIScripts/Common/Invoke-CommandWithFunctions.ps1

```diff
@@ -0,0 +1,24 @@
+. $PSScriptRoot/Aliases.ps1
+function Invoke-CommandWithFunctions {
```

Comment:
Can you, please, add a comment describing purpose of this function (including problems related to invoking functions in remote session)? Either like Invoke-UntilSucceeds function documentation or like Invoke-NativeCommand comment explanation.